### DISCUSSION TITLE
Searching 'Area' in options should show all tablet options

### DISCUSSION THREAD

**Discussion by JamieWright99** [ppy/osu#23504] [Ideas] [answered]
Currently when typing area into the settings not many tablet options actually show up, you cant even change the tablets width/height settings to create the area. It would be a nice quality of life change if searching for area let you resize the area. 

Here's a comparison between the options that show typing 'area' vs 'tablet'
![Area Lazer](https://github.com/ppy/osu/assets/102170503/7efe86fb-506b-4b02-8d9b-0a4e41adc94d)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23506
